lib/hitobito_sac_cas/wagon.rb

```diff
@@ -109,10 +109,9 @@ class Wagon < Rails::Engine
         "people.id" => SearchStrategies::SqlConditionBuilder::IdMatcher,
         "people.birthday" => SearchStrategies::SqlConditionBuilder::BirthdayMatcher
       )
-      # SearchStrategies::Sql::SEARCH_FIELDS['Person'][:attrs] << 'people.id'
```

Comment:
old sphinx code

Reply:
LGTM you just have to fix the tests and then we're good to go :racing_car: